.github/workflows/firebase-deployment.yml

```diff
@@ -18,6 +18,9 @@ jobs:
           node-version-file: '.nvmrc'
           cache: 'npm'
 
+      - name: Install NPM packages
+        run: npm ci
+
       - name: Gatsby Cache Folder
```

Comment:
should restoring the cache happen before installing deps?

Reply:
Facepalm to me ;D

Reply:
Actually correct. The cache restore after is the gatsby one